lib/PlacesProvider.js

```diff
@@ -475,9 +475,9 @@ Links.prototype = {
 /**
  * Singleton that serves as the default link provider for the grid.
  */
-const gLinks = new Links();
+const gLinks = new Links(); // eslint-disable-line no-use-before-define
```

Comment:
I don't really feel like  `no-use-before-define` is necessary anymore since `const`/`let` throw `ReferenceError`s if they're defined after usage, unlike `var`, so I wouldn't be opposed to leaving out this rule